### ISSUE TITLE
[docs] Slug based linked headlines in volto-slate

### ISSUE THREAD

**Issue by stevepiercy** [plone/volto#5755]
@nileshgulia1 @tiberiuichim @avoinea @sneridagh I am currently writing the roadmap for Plone 6.x and I try to include links to the documentation of a new feature. So far I only found:

https://6.docs.plone.org/volto/configuration/volto-slate/index.html
https://6.docs.plone.org/volto/configuration/volto-slate/configuration-settings.html#slate-uselinkedheadings

Is this everything we got/need?

We could use some narrative docs in the User Manual, specifically [Text block](https://6.docs.plone.org/volto/user-manual/blocks.html#text-block).

_Originally posted by @tisto in https://github.com/plone/volto/issues/4298#issuecomment-1939162027_

**Comment by praveen-rikhari** [comment]
Hey @stevepiercy can I work on this? To understand things better is this about adding the slug based links docs to be added to user manual as well??

**Comment by stevepiercy** [comment]
@praveen-rikhari please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), [Contributing to Plone](https://6.docs.plone.org/contributing/index.html), and [Contributing to Volto](https://6.docs.plone.org/contributing/volto.html).

For this item, because it is docs only, I would recommend setting up docs following [Building and checking the quality of documentation](https://6.docs.plone.org/contributing/documentation/setup-build.html).

To take screenshots, you can use either https://demo.plone.org/ or install Plone locally.

Finally note that I updated the issue description with the link to where new docs need to be added.

**Comment by praveen-rikhari** [comment]
Hey @stevepiercy As this would be my first contribution to the project i have already submitted the CLA signing process, i am waiting on it meanwhile i will try to prepare this locally till then

**Comment by stevepiercy** [comment]
I don't know if you can solve this issue, because I don't know your abilities. 😏 

@Aryan285git please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do), [Contributing to Plone](https://6.docs.plone.org/contributing/index.html), and [Contributing to Volto](https://6.docs.plone.org/contributing/volto.html).

**Comment by Tushar9650** [comment]
Could I Work on it Now ?? @stevepiercy

**Comment by stevepiercy** [comment]
@Tushar9650 please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do), [Contributing to Plone](https://6.docs.plone.org/contributing/index.html), and [Contributing to Volto](https://6.docs.plone.org/contributing/volto.html).

**Comment by harz05** [comment]
hello @stevepiercy, im new to contributing so this question may seem silly but what exactly am I required to do? From what I understood I only need to update the documention for the text block content and write how it could be used(similar to how it explained for other blocks in the docs).

**Comment by stevepiercy** [comment]
@harz05 correct, we need only documentation for the User (Editor) Manual for the [Text block](https://6.docs.plone.org/volto/user-manual/blocks.html#text-block) for how to add a title or subtitle so that an anchor is also automatically generated and shown for authenticated users only. We should tell Editors to ask their Site Administrator (or Developer) to enable automatic anchors for their site for anonymous users, if they want that.

Note that the Text block includes all the features of Slate, including formatting (bold, italic, and strikethrough, title, subtitle, subscript, superscript, numbered list, bulleted list, and blockquote) and link (add or use existing content type). Those other items could also be documented as part of this issue, and help complete:

- https://github.com/plone/volto/issues/3827

The two links to the Slate API and configuration are already documented, and are meant only as a reference for Developers and Admins, not Editors. However Editors might want to talk to their Developer to change the default anchor display setting that I mentioned above.

Hope that helps. Pleas let me know. Thank you!

**Comment by harz05** [comment]
thanks a lot @stevepiercy, i've started working on it and will raise a PR once it's done to get it reviewed
